src/twisted/internet/test/reactormixins.py

```diff
@@ -324,26 +326,30 @@ def stop():
             timedOutCall.cancel()
 
 
-    def makeTestCaseClasses(cls):
+    @classmethod
+    def makeTestCaseClasses(
+        cls: Type['ReactorBuilder']
+    ) -> Dict[str, Union[Type['ReactorBuilder'], Type['SynchronousTestCase']]]:
         """
         Create a L{SynchronousTestCase} subclass which mixes in C{cls} for each
         known reactor and return a dict mapping their names to them.
         """
-        classes = {}
+        classes = {}  # type: Dict[str, Union[Type['ReactorBuilder'], Type['SynchronousTestCase']]]   # noqa
         for reactor in cls._reactors:
             shortReactorName = reactor.split(".")[-1]
-            name = (cls.__name__ + "." + shortReactorName + "Tests").replace(".", "_")
-            class testcase(cls, SynchronousTestCase):
+            name = (cls.__name__ + "." + shortReactorName +
```

Comment:
I guess twistedchecker complained at you?

Reply:
Yes, I made a lot of small changes like this to appease twistedchecker in terms of line spacing, line length, etc.